components/atom/label/src/index.scss

```diff
@@ -6,14 +6,29 @@ $c-atom-label-type: success $c-success, error $c-error;
 
 .sui-AtomLabel {
   display: block;
-  font-size: $fz-base;
```

Comment:
Shouldn't the field have a default font-size?

Reply:
```
&--medium {
    font-size: $fz-base;
  }
``` is the default one

Reply:
IMHO Then it is not a "modifier", but the default font-size and it should be in the main definition